companies/american-alpine-club.md

```diff
@@ -0,0 +1,21 @@
+---
+name: American Alpine Club
+image: american-alpine-club.png
+blm_statements:
+  - url: https://twitter.com/americanalpine/status/1267946389482266625
+    date_posted: 2020-06-02
+    type: Tweet
+    raw_text: >
+     Black lives matter—throughout the country and at the crag. The AAC stands in solidarity with those protesting systemic racism on the frontlines and from home. 
+  - url: https://www.facebook.com/AmericanAlpineClub/photos/a.310680388934/10157980875738935/
+    date_posted: 2020-06-02
+    type: Blog
+    raw_text: >
+     Black lives matter—throughout the country and at the crag. The American Alpine Club stands in solidarity with those protesting systemic racism on the frontlines and from home. As climbers and as humans, we believe that racism has no place in our craft or our country.
+     
+     Please read our full statement, "The AAC Stands in Solidarity."
+     
+     https://bit.ly/2BmOZFf
+exec_team_url: https://americanalpineclub.org/staff
+diversity_inclusion_url: https://americanalpineclub.org/news/2020/6/2/the-aac-stands-in-solidarity?fbclid=IwAR0eNytmunD0H0s915uGahQA069P4UxdTeMU14a7Q_uo0ELZ0kMTOxjE-Wc
```

Comment:
```suggestion
diversity_inclusion_url: https://americanalpineclub.org/news/2020/6/2/the-aac-stands-in-solidarity
```

It looks like we can use this URL without the extra query params. I need to add this step into the guide.

Reply:
Url changed to reflect deletion of query params, please see if I did it correctly

Reply:
yep that's perfect

Reply:
yamllint errors fixed, can it be merged with main?

Reply:
I'm still getting this one pesky error about the wrong syntax on line 10, can you check to see if I overlooked something?